src/L.Control.Heightgraph.js

```diff
@@ -152,7 +169,7 @@ L.Control.Heightgraph = L.Control.extend({
      */
     _calculateFullExtent(data) {
         if (!data || data.length < 1) {
-            throw new Error("no data in parameters");
+            return null;
```

Comment:
When running in embed mode and always expanded, the data array could be empty. Instead of erroring out, the heightgraph would still work by rendering nothing.